### ISSUE TITLE
Avatar in notification

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#8]
```
In the notification we can see the Pino icon. I think that it would be
great to see the avatar of the person who tweeted. Maybe some kind of
avatar cache is needed?

Great job
Cheers
```

Original issue reported on code.google.com by `bicode...@gmail.com` on 14 Dec 2009 at 4:07

**Comment by GoogleCodeExporter** [comment]
```
This is for 0.2.0 release. See in TODO list in the bottom of main page.

> Userpics in notifications
```

Original comment by `tro...@gmail.com` on 14 Dec 2009 at 11:21
- Changed state: **Accepted**
- Added labels: **Priority-Low**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
I knew this wasn't forgotten. Thank you troorl
```

Original comment by `bicode...@gmail.com` on 14 Dec 2009 at 2:53

**Comment by GoogleCodeExporter** [comment]
```
Closed by ef39ca374f revision
```

Original comment by `tro...@gmail.com` on 2 Jan 2010 at 6:36
- Changed state: **Fixed**
